developer_guides/virtualisation/virtualisation.rst

```diff
@@ -0,0 +1,255 @@
+.. _virtualisation:
+
+SOF VirtIO design
+#################
+
+.. contents::
+
+1. General information
+**********************
+
```

Comment:
I would add something here describing the feature. i.e. what does this code do and why it is useful.

Reply:
Please put 
```
This document describes the Linux driver architecture to virtualise SOF DSP firmware features across multiple guest VMs.

It describes a 

1) SOF VirtIO (Virtual Frontend) driver to be run on a guest virtual machine.
2) SOF vhost (Virtual Backend) driver to be run on the host.

This driver architecture allows full SOF audio use cases to reside on either guest VM or host. It also allows non audio DSP firmware features to also be virtualised on guests.
```

Reply:
not seeing this, have you pushed ?

Reply:
@lgirdwood yes, it's there. It now has 4 entries instead of 2, maybe that's why you didn't notice it?